documentation_builder/simulating.ipynb

```diff
@@ -758,13 +759,14 @@
       "PRODUCING REACTIONS -- Pyruvate\n",
       "-------------------------------\n",
       "  %             FLUX   RXN ID                        REACTION                       \n",
-      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
-      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
+      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
```

Comment:
Exactly, the percentages are calculated from the FBA solution. This is probably another item to add to #240.

Its non-trivial to decide how percentages should be assigned in the FVA case. Mainly because the total flux through each node is no longer fixed, and the percentages will necessarily change. 

Finding errors in percentages would probably also be too hard, although that would be a cool feature.

Perhaps they should just not be printed to avoid confusion.